src/vendor/Template.js

```diff
@@ -15,15 +21,15 @@ function expandLoop(loop, namespaces) {
   const { from, template, separator } = loop;
 
   if (!isString(from)) {
-    Log.error('expecting from clause to be string');
+    Template.log.error('expecting from clause to be string');
```

Comment:
The `logs.js` and `Template.js` imported each other.  Now, I will have `logs.js` import `Template.js` and then set `Template.log` to the `Log` object